src/main/java/com/google/research/bleth/simulator/AbstractSimulation.java

```diff
@@ -0,0 +1,33 @@
+package com.google.research.bleth.simulator;
+
+/** A placeholder class for representing a BLETH simulation. */
+public abstract class AbstractSimulation {
+
+    /** A placeholder class for representing a BLETH simulation builder. */
+    public static abstract class Builder {
+
+        public abstract String getSimulationType();
+
+        public int getMaxNumberOfRounds() { return 1; }
```

Comment:
if this is an abstract class then you should not implement pure abstract methods (that is, do not "return 1")

Reply:
This `AbstractSimulation` is a placeholder, and it exists only to provide a mock simulation in tests. This implementation will be overridden as soon as PR #6 is merged to master.